docs/README-Extending.md

```diff
@@ -210,6 +210,35 @@ Notice the syntax we use for the context value (or `viewItem`) above is a regula
 
 Extenders can define command groups separated by dividers in VS Code's right-click context menus by using the `group` property for items in `contributes.menus.view/item/context` of their `package.json`. Zowe Explorer prefixes its menu command `group` values with `0##_zowe_`, where `0##` represents numbers 000 - 099. Thus, extenders should avoid using `0##_zowe_` at the beginning of any `group` values they assign for menu commands. Any command groups contributed by extenders should be located below Zowe Explorer's command groups whenever they appear together in the same context menu. This helps keep the core Zowe Explorer context menu commands in a uniform location for users. A recommended extender menu command `group` naming convention would be to prefix the extender's `group` values with `1##_extensionName_`.
 
+### Accessing Zowe Explorer tree item information
+
+When adding a command to the right-click context menu of an item in Zowe Explorer's Data Sets, USS, or Jobs tree view, extenders may want to access information specific to that item (or "node"). This can be done by importing that item's node type, and then specifying that same node type when defining the node as a parameter for the command's callback function. The node types for items in each of Zowe Explorer's three tree views are as follows:
+
```

Comment:
I would maybe move the list of node types (starting from "The node types for items...") under the code example, so that the general explanation ("When adding a command...") and the example for the explanation ("In the example below...") are together. It seems a bit confusing/non-linear to start with an explanation, put a list in the middle, and then add the example for the explanation later on.

I think I wouldn't understand the explanation in the first paragraph if I didn't know what the code in ZE looks like, and the node types that are available are kind of useless unless I know what to use them for (after lookingat the code sample).

Also I def agree with @phaumer that more examples would be cool :)

Reply:
@katelynienaber Thanks for this suggestion! I've rearranged the information accordingly, and reworded it a bit to flow more smoothly in its new location.